### PR TITLE
Fix: Add chrome launcher flags for Docker

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

File: packages/connector-chrome/src/chrome-launcher.ts [hunk 1]
```diff
@@ -132,6 +132,8 @@ export class CDPLauncher extends Launcher {
             /* istanbul ignore next */
             if (isCI) {
                 this.chromeFlags.push('--headless', '--disable-gpu');
+            } else if (process.env.DOCKER === 'true') { // eslint-disable-line no-process-env
+                this.chromeFlags.push('--headless', '--disable-gpu', '--no-sandbox');
```
Comment (alrra) [inline packages/connector-chrome/src/chrome-launcher.ts]: @sarvaje Can you provide a link or more details about `--no-sandbox`? Thanks!

Comment (alrra) [reply packages/connector-chrome/src/chrome-launcher.ts]: Never mind

Comment (alrra) [inline packages/connector-chrome/src/chrome-launcher.ts]: `--no-sandbox`

Ref: https://developers.google.com/web/updates/2017/04/headless-chrome#faq

**Comment by alrra** [comment]
@sarvaje Change included in `@hint/connector-chrome` `v1.1.0` 🎉

File: packages/connector-chrome/src/chrome-launcher.ts (hunk 1 above)
Comment (molant) [inline packages/connector-chrome/src/chrome-launcher.ts]: Do we need `--disable-gpu` if we are in a linux container?  In the [FAQ](https://developers.google.com/web/updates/2017/04/headless-chrome#faq) it looks like is only for Windows

**Pull request closed without merging**